src/ducks/files/Toolbar.jsx

```diff
@@ -29,6 +30,8 @@ class Toolbar extends React.Component {
 
   render () {
     const { t, disabled, displayedFolder, actions, onSelectItemsClick, uploadFiles } = this.props
+    const notRootfolder = displayedFolder !== undefined && displayedFolder !== null && displayedFolder.dir_id === ROOT_DIR_ID ? true : false
```

Comment:
That is more or less the message travis gave us:

![][(http://i.imgur.com/FgxMouX.png)